Updates the Cow.Fi website text.

avsavsavs opened the pull request:
This commit updates the website according to what we agreed to roll out with. The original text that was agreed and reviewed can be found here: https://docs.google.com/document/d/1jNt2Wp31sVBw5FLA5neubxgqd_dadx5ETXbUvLoPrQY/edit. 

Hopefully, I have not made any edit mistakes.

anxolin commented on pages/index.tsx:
```diff
@@ -69,21 +69,21 @@ export default function Home({ metricsData, siteConfigData }) {
               <span>
                 <b>Superior on-chain DeFi liquidity</b>
-                <p>All on-chain liquidity sources are continiously integrated which allows for getting the best on-chain prices.</p>
+                <p>In order to give the best trading prices, existing and new liquidity sources are continuously being integrated.</p>
```
This texts is strange "existing"

anxolin reviewed (commented):
Nice first version!

I'm still not super convinced about the messages, but this makes a good first version, so in this sense I think I'm ready to approve.

In the future we should simplify (but is okay now I guess). Some other folks that simplified a lot their language and their content got better and might have a similar audience as us is https://ethereum.org/en/

Also, I believe this message is wrong:

![image](https://user-images.githubusercontent.com/2352112/147974196-a73cc3b4-472f-497f-93a5-edc274447e0e.png)

`JIT (Just in time) prices`, something in the lines of:

The liquidity used to find you the best price uses the best path of execution. However, one difference between CoW protocol and other exchanges is that, in CoW Protocol the best path is calculated right before the trade is executed instead of when the order is created. This make it superior for Smart Contract integrations. 

Other than that, I would approve/merge once you address the comments.

annamsgeorge commented:
Hi, overall love it but would suggest some text changes:


1) Batch Settlement layer for MEV protected trades --> The Settlement Layer for decentralized trading
1b) I don't think we need the "scroll down" button, users know they can scroll down and it looks a bit too crowded. 
2) Getting you better prices, MEV Protection,
no failed txs and gas costs savings through the batch settlemebt execution model. --> (no failed tx is incorrect, just that the user doesn't pay for it)
Trade on Cow Protocol for better prices, gas cost savings, and extra secure MEV protection. 
3) Trade volume: $4B+ 
All time trades: 165K+
4) I would not advertise fees we are taking, from the user perspective we want to show that we are providing good prices, why then would we advertise how much fees we are collecting (I know Martin sometimes mentioned it's a good metric to show the value the token has, I am somewhat sceptical though to advertise fees yet, anyways we are not making more money than we are spending. Even Uniswap, for whom this metric is really positive and useful for their LP providers don't put it on their website. I really wouldn't put it.) 
There's a way we could express this more positively:
Option a) 
$8.6M ETH saved in tx costs. (This is mentioning the amount of fees we have been taking, but rather than phrasing it as fees captures by us, we are mentioning that users didn't have to spend their precious ETH on tx costs. I understand if you say this is too confusing though.)
Option b)  A better metric would be surplus generated for users. 
Option c) We substract the total fees collected from the total fees we paid to subsidies the solvers and display this amount saying "Total fees saved for users".
Let me know which one you like best please, I prefer option b, although the problem there is that we can't show the real surplus but only an estimate of the difference between limit price and received price. 

(Will continue with further feedback in next comment)

annamsgeorge commented:
More than a Meta Dex Aggregator:

Would not start with uniform clearing price, not the main thing users care about. I would prefer to stick to something similar I had shared previously or understand why this was not used? 
We should aim to keep the language as simple as possible. I am not sure where this JIT is coming from / I don't think we should introduce even more new language users don't understand. 


1) BEST ON-CHAIN PRICES
By leveraging any on-chain liquidity sources we are able to provide the best on-chain prices. 
2) PROFESSIONAL TX EXECUTION
Meta-transactions ensure that users don’t pay for failed transactions. They also facilitate automatic resubmission of failed orders. 
3) COINCIDENCE OF WANTS (CoWs)
CoWs, simple put as peer-to-peer trades, improve the prices by reducing the need for liquidity pools. This saves the users money spent on protocol fees, slippage, and gas fees.
4) MEV MITIGATION 
Trades are protected against MEV in multiple layers: via Coincidence of Wants and leveraging both Flashbots Protect RPC and the Eden Network.  
5) Same Block, Same Price
All trades within the same block are guaranteed to receive the same price regardless of their execution order.
6) ETH-less trading experience
Users pay fees in their sell or buy token. No need to spend your precious ETH.

annamsgeorge commented:
1) The protocol improves prices for users by batching trades, finding coincidence of wants (CoWs)
and tapping into all on chain liquidity - including aggregators. --> We should have one unified way of spelling Coincidence of Wants (capital C, small o, capital W)
2) Your guardian in the dark forest section: I don't understand why we are repeating advantages of cowswap here that are already listed above? Would remove this section, we can keep the graphic though.

annamsgeorge commented:
Re the third number we could list (surplus), we can use Ben's metric of only counting unusual large suprlus provided to users (more than 10% of their trade size). 
Total surplus given to users: $3M+

(taken from https://dune.xyz/gnosis.protocol/GPv2-Trader-Surplus)

anxolin committed:
Updates the website text.

This commit updates the website according to what we agreed to roll out with. The original text that was agreed and reviewed can be found here: https://docs.google.com/document/d/1jNt2Wp31sVBw5FLA5neubxgqd_dadx5ETXbUvLoPrQY/edit.

Hopefully, I have not made any edit mistakes.

anxolin committed:
Update index.tsx

anxolin commented:
Hei Alex! 

I rebased your branch with the latest and resolved the conflicts. You can apply annas changes and review Haris comments?

https://gnosisinc.slack.com/archives/CPZA1AGKY/p1641374572116800?thread_ts=1641239634.062200&cid=CPZA1AGKY


I will take this one too from the list of Anna's `3` and `4.` (related to metrics)

anxolin commented:
@avsavsavs im merging this to consolidate, pls apply the changes to a new PR
